app/view_models/profile_index.rb

```diff
@@ -62,10 +62,10 @@ def manage_personal_key_partial
   def header_personalization
     return decrypted_pii.first_name if decrypted_pii.present?
 
-    current_user.email
+    decorated_user.email
```

Comment:
Ah good point.

I'm used to decorators that have explicit delegation, so I'd expect `delegate :email, to: :model` or something